src/website/shared/management/commands/propose_cve_links.py

```diff
@@ -0,0 +1,38 @@
+import argparse
+import datetime
+import logging
+from typing import Any
+
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+from shared import models
+from shared.listeners.automatic_linkage import build_new_links
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Propose new CVE links on all CVE of a certain time range"
+
+    def add_arguments(self, parser: argparse.ArgumentParser) -> None:
+        parser.add_argument(
+            "delta_range_in_days",
+            help="Since when should we redo all the links in days? e.g. `5` for 5 days ago.",
+        )
+
+    def handle(self, *args: Any, **kwargs: Any) -> None:
+        _delta = kwargs["delta_range_in_days"]
+
+        try:
+            delta = datetime.timedelta(days=int(_delta))
+            since_date = datetime.datetime.now() - delta
+        except ValueError:
+            raise CommandError(f"Not a valid delta format: {_delta}")
```

Comment:
How can this go wrong except for the cast?

Reply:
What happens if you put 365 * 1_000_000 years? :-)

Reply:
I even fixed that, but I think we can keep the defensive programming :).